core/src/main/scala/com/rackspace/com/papi/components/checker/ByteArrayServletInputStream.scala

```diff
@@ -13,14 +13,24 @@
  *   See the License for the specific language governing permissions and
  *   limitations under the License.
  */
-package com.rackspace.com.papi.components.checker.servlet
+package com.rackspace.com.papi.components.checker
```

Comment:
This was a combination of two copy/pasta classes. One originally lived in validator-base.scala (com.rackspace.com.papi.components.checker) and the other in inputStream.scala (com.rackspace.com.papi.components.checker.servlet). I just added the extra constructor and squashed a class.

``` scala
   def this(content : String) {
    this(content.getBytes())
  }
```